test/OpenSpaceToolkit/Physics/Environment/Atmospheric/Earth.test.cpp

```diff
@@ -207,6 +209,20 @@ TEST(OpenSpaceToolkit_Physics_Environment_Atmospheric_Earth, GetDensityAt_LLA)
         const LLA lla = LLA(Angle::Degrees(35.076832), Angle::Degrees(-92.546296), Length::Kilometers(1001.0));
         const Instant instant = Instant::J2000();
 
-        EXPECT_ANY_THROW(earthAtmosphericModel.getDensityAt(lla, instant));
+        const std::string expectedString = "Exponential density model is not valid for altitudes above 1000 km. Altitude = 1001000.0 [m]";
```

Comment:
```suggestion
        const String expectedString = "Exponential density model is not valid for altitudes above 1000 km. Altitude = 1001000.0 [m]";
```

Reply:
Ah man I had convinced myself that this didn't work, but just tried again and seems to be fine. Changed 👍